src/bindings/gsettings/CMakeLists.txt

```diff
@@ -50,15 +50,34 @@ if (GELEKTRA_LIBRARY)
 		install (TARGETS elektrasettings LIBRARY DESTINATION ${GIO_MODULE_DIR})
 	endif ()
 else()
-	pkg_check_modules (GELEKTRA gelektra-4.0>=0.8.16 QUIET)
-	if (!GELEKTRA_FOUND)
-		remove_binding (glib "elektra glib bindings needed for gsettings backend")
-	endif()
+	pkg_check_modules (GELEKTRA gelektra-4.0>=0.8.16 REQUIRED)
 	include_directories (${GELEKTRA_INCLUDE_DIRS})
 	target_link_libraries (elektrasettings ${GLIB_LIBRARIES} ${GMODULE_LIBRARIES} ${GIO_LIBRARIES} ${GELEKTRA_LIBRARIES})
 	install (TARGETS elektrasettings LIBRARY DESTINATION ${GIO_MODULE_DIR})
 endif ()
 
+if (BUILD_TESTING)
+	if (GLIB_VERSION VERSION_LESS 2.48)
+		message (WARNING "excluding GSettingsTests: requires GLib >= 2.48")
+	else ()
+	include (ExternalProject)
+	ExternalProject_Add (GSettingsTests GIT_REPOSITORY https://github.com/sirblackheart/gsettings-tests INSTALL_COMMAND "")
```

Comment:
The repo should be at least at ElektraInitiative.

Is there no way to use the official repo? I would prefer it to have as normal dependency via package manager, not as CMake ExternalProject. What about building modified  GSettingsTests debian packages?